### ISSUE TITLE
Feature request: Automatically interpret lyrics as synced if they are formatted that way

### ISSUE THREAD

**Issue by rexendevar** [Taiko2k/Tauon#1340]
I know Tauon doesn't detect synced lyrics unless there is a .lrc file beside them, but the music player I use on my phone is able to automatically interpret the standard lyrics field as synced. 

![screenshot2024-12-08_14-31-48](https://github.com/user-attachments/assets/492cb9db-bca2-4943-9edf-e8e293920c82)
(clicking show synced lyrics does nothing)
compare to my mobile player:
![Screenshot_20241208-143553](https://github.com/user-attachments/assets/ae4bbbd2-bc47-4698-aa29-649e558af701)

basically just run a test on the content of the lyrics field and pass it over to the .lrc display code if it is formatted the same way. SHOULD be pretty simple i think but i don't know these things. apologies for the screenshots being enormous i don't know how to make them smaller :+1:

**Comment by C0rn3j** [comment]
> apologies for the screenshots being enormous i don't know how to make them smaller 👍

It's a bit awkward - https://gist.github.com/stevecondylios/dcadb4fc73e63f27a3bbcf17e52058bf

**Comment by NeuHix** [comment]
+1
